### DISCUSSION TITLE
Azure Route Server module available in CLZ CAF library

### DISCUSSION THREAD

**Discussion by ruchisinghal01** [aztfmod/terraform-azurerm-caf#996] [Q&A]
Any idea if Azure Route Server module available in CLZ CAF library or if it is development phase?

**Comment by arnaudlh** [comment]
Yes @ruchisinghal01, this PR: https://github.com/aztfmod/terraform-azurerm-caf/pull/1006 has been merged and will be up for 5.5.5

**Comment by Smago** [comment]
Hello,

We are trying to deploy ARS in our environment but we face 2 errors: 

1. Apparently we cannot deploy ARS in regions that are not Availability Zone Compatible and we did not find any parameters (maybe we misunderstood something) 
2. How can I configure BGP peering with CAF ? we want configure BGP Peering to ours NVA for BGP communications

Thx for you support
